test/cppsuite/test_harness/workload_generator.h

```diff
@@ -142,7 +142,9 @@ class workload_generator : public component {
     void
     set_tracker(workload_tracking *tracking)
     {
-        _enable_tracking = (tracking != nullptr);
+        /* Tracking cannot be NULL. */
+        testutil_check(tracking == nullptr);
```

Comment:
Is this backwards?

Reply:
testutil_check will exit the program if the returned value is not 0